openshift/templates/email-verification-service/email-verification-service-deploy.yaml

```diff
@@ -42,27 +90,45 @@ objects:
       name: ${NAME}
       weight: 100
 
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: ${NAME}-email-host
+    labels:
+      app: ${APP_NAME}${SUFFIX}
+      name: ${NAME}${SUFFIX}
+      app-group: ${APP_GROUP}
+      role: ${ROLE}
+      env: ${TAG_NAME}
+  stringData:
+    email-host: ${BCGOV_EMAIL_HOST}
```

Comment:
I would name the environment variable just `SMTP_SERVER_HOST` as this *may* be used outside of BCGov as well.

Reply:
Agreed, avoid implementation specific naming conventions.

Reply:
changed to SMTP